codegen.yml

```diff
@@ -0,0 +1,23 @@
+overwrite: true
+schema: "http://localhost:8080/graphql"
+documents: './src/app/graphql/queries.ts'
+generates:
+  ./src/app/graphql/types.tsx:
+    plugins:
+      - add:
+          content:
+          - ''
+          - 'export namespace GraphQL {'
+      - add:
+          placement: 'append'
+          content: '}'
+      - "typescript"
+      - "typescript-operations"
+      - "typescript-react-apollo"
+    config:
+      withHOC: false
+      withHooks: true
```

Comment:
I think we should avoid to use generated hooks due to below reasons.

1. Every time we need re-generate entities after changing gql query
2. Later we will implement error handling and we will bound to add error handling for each query. But we will handle all the errors in custom hook on top of useQuery and useMutation i.e. generically 

The main purpose of generating entities is to implement type safety and for that we need types

Reply:
@ajaypratap003 Honestly, Then there is no point in using this codegen